packages/shared/package.json

```diff
@@ -17,14 +17,6 @@
     "pre-commit": "echo 'No pre-commits defined in: shared'",
     "start": "tsc --build --clean && tsc --build && tsc --build tsconfig.json"
   },
-  "devDependencies": {
-    "@babel/core": "^7.22.10",
-    "@types/node": "^20.4.8",
-    "@types/react": "^18.2.19",
-    "babel-loader": "^9.1.3",
-    "eslint-config-prettier": "^8.10.0",
-    "prettier": "^3.0.1"
-  },
```

Comment:
@chybisov Just removed that complete devDependenies-stack from the shared package.json. It works just fine locally, do you see issues with that? (As you pointed out, we might be left with just a couple ones but here we got rid of it all)

Reply:
@dennyscode, that's fine. Can you please try to remove some from dapp folder?

Reply:
I got to remove the "emotion" packages from dapp and shared folder, too. Not sure, if there´s more to be removed but that´s where we are now.